### ISSUE TITLE
ListView can't hold directives alone in template

### ISSUE THREAD

**Issue by diestrin** [NativeScript/nativescript-angular#93]
I have the following code
```
@Component({
  selector: 'TodoList',
  directives: [Todo],
  template: `
<ListView [items]="todos">
  <item-template>
    <template #item="item">
      <Todo [item]="item"></Todo>
    </template>
  </item-template>
</ListView>
`
})
```

But when it tries to render the todos, I got the following error `file:///app/tns_modules/ui/core/view.js:107:116: JS ERROR Error: onMeasure() did not set the measured dimension by calling setMeasuredDimension()`

However, if I wrap the `<Todo>` in a `<StackLayout>`, it works.

Not sure what's exactly going on, but what I asume, is that `<Todo>` (or any other custom directive) doesn't have a default height/width, so that's why it fails to render.

**Comment by hdeshev** [comment]
Thanks for the report. I confirmed the issue, and it's caused by the ProxyViewContainer element that gets inserted for components. It's an easy fix for Android, but we'll need some time to take care of it for iOS.

**Comment by hdeshev** [comment]
The iOS fix landed in master in the tns-core-modules repository. Unfortunately, releasing an unofficial build there is a bit tricky at the moment, as it requires a new release in the Android runtime.

The fix will go live with tns-core-modules@1.7 which is due next week.

**Comment by diestrin** [comment]
Great news man, thanks :)
